docs/tutorials/HPCIntro.md

```diff
@@ -141,12 +141,12 @@ backend.providers.SGE.config {
 When Cromwell runs a task, it will fill in a template for the job using the declared runtime attributes. This specific template will vary depending on the requirements of your HPC cluster. For example, say you normally submit jobs to SGE using:
 
 ```bash
-qsub -terse -V -b n -N my_job_name \
+qsub -terse -V -b y -N my_job_name \
   -wd /path/to/working_directory \
   -o /path/to/stdout \
   -e /path/to/stderr \
   -pe smp 1 -l mem_free=0.5g -q short \
-  myScript.bash
+  bash myScript.bash
```

Comment:
I would be consistent with either `bash` or `/bin/bash` everywhere, unless there's a reason not to ?

Reply:
Yes I think you are right, I know some arguments call for '/usr/bin/env bash', which allows a separate version of bash to be specified with environment variables. I think that might be the best way.